File: behavioral/java/state/src/USDAccount.java
public class USDAccount implements AccountA{
    @Override
    public void withdrawMoney(int moneyAmount) {
        System.out.println("Withdraw money of USD account. " + Integer.toString(moneyAmount) + "USD");
    }

    @Override
    public void depositMoney(int moneyAmount) {
        System.out.println("Deposit money of USD account. " + Integer.toString(moneyAmount) + "USD");
    }

    @Override
    public String getAccountName() {
        return "USD";
    }
}

File: behavioral/python/state/USDAccount.py
from AccountA import AccountA

class USDAccount(AccountA):
    def withdrawMoney(self, moneyAmount: int) -> None:
        print("Withdraw money of USD account. " + str(moneyAmount) + "USD")
    def depositMoney(self, moneyAmount: int) -> None:
        print("Deposit money of USD account. " + str(moneyAmount) + "USD")
    def getAccountName(self) ->str:
        return "USD"